test/f-promise-test.ts

```diff
@@ -124,4 +124,26 @@ describe(module.id, () => {
         strictEqual(queue.read(), 9);
         strictEqual(queue.peek(), undefined);
     });
+
+    describe('collection functions', (done) => {
+        it('map', () => {
```

Comment:
`done` should be the `it` callback rather than the `describe` callback, shouldn't it?